@navikt/core/tokens/darkside/create-configuration.ts

```diff
@@ -0,0 +1,61 @@
+import { getGlobalScaleForColor } from "./leonardo";
+import { contrastTokenConfig } from "./tokens/contrast";
+import { neutralTokenConfig } from "./tokens/neutral";
+import { radiusTokenConfig } from "./tokens/radius";
+import { semanticTokenConfig } from "./tokens/semantic";
+import { semanticTokensForAllRolesConfig } from "./tokens/semantic-roles";
+import { spacingTokenConfig } from "./tokens/spacing";
+import {
+  ColorThemeMode,
+  GlobalColorVariable,
+  globalColorRoles,
+  mergeConfigs,
+  tokensWithPrefix,
+} from "./util";
+
+export const globalScale = (
+  mode: ColorThemeMode,
```

Comment:
What do you think about "theme" instead of "mode"?
```suggestion
  theme: ColorTheme,
```

Reply:
Sure 👍

Reply:
Done ⚡